diff -Naur lollypop-0.9.42.orig/src/application.py lollypop-0.9.42/src/application.py
--- lollypop-0.9.42.orig/src/application.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/application.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/art.py lollypop-0.9.42/src/art.py
--- lollypop-0.9.42.orig/src/art.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/art.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/art_album.py lollypop-0.9.42/src/art_album.py
--- lollypop-0.9.42.orig/src/art_album.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/art_album.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/art_base.py lollypop-0.9.42/src/art_base.py
--- lollypop-0.9.42.orig/src/art_base.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/art_base.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # Copyright (c) 2013 Vadim Rutkovsky <vrutkovs@redhat.com>
 # Copyright (c) 2013 Arnel A. Borja <kyoushuu@yahoo.com>

diff -Naur lollypop-0.9.42.orig/src/art_radio.py lollypop-0.9.42/src/art_radio.py
--- lollypop-0.9.42.orig/src/art_radio.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/art_radio.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/cellrendereralbum.py lollypop-0.9.42/src/cellrendereralbum.py
--- lollypop-0.9.42.orig/src/cellrendereralbum.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/cellrendereralbum.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/codecs.py lollypop-0.9.42/src/codecs.py
--- lollypop-0.9.42.orig/src/codecs.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/codecs.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/collectionscanner.py lollypop-0.9.42/src/collectionscanner.py
--- lollypop-0.9.42.orig/src/collectionscanner.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/collectionscanner.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/container.py lollypop-0.9.42/src/container.py
--- lollypop-0.9.42.orig/src/container.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/container.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/database.py lollypop-0.9.42/src/database.py
--- lollypop-0.9.42.orig/src/database.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/database.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/database_albums.py lollypop-0.9.42/src/database_albums.py
--- lollypop-0.9.42.orig/src/database_albums.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/database_albums.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/database_artists.py lollypop-0.9.42/src/database_artists.py
--- lollypop-0.9.42.orig/src/database_artists.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/database_artists.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/database_genres.py lollypop-0.9.42/src/database_genres.py
--- lollypop-0.9.42.orig/src/database_genres.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/database_genres.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/database_tracks.py lollypop-0.9.42/src/database_tracks.py
--- lollypop-0.9.42.orig/src/database_tracks.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/database_tracks.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/define.py lollypop-0.9.42/src/define.py
--- lollypop-0.9.42.orig/src/define.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/define.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/fullscreen.py lollypop-0.9.42/src/fullscreen.py
--- lollypop-0.9.42.orig/src/fullscreen.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/fullscreen.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/inotify.py lollypop-0.9.42/src/inotify.py
--- lollypop-0.9.42.orig/src/inotify.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/inotify.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/lastfm.py lollypop-0.9.42/src/lastfm.py
--- lollypop-0.9.42.orig/src/lastfm.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/lastfm.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/list.py lollypop-0.9.42/src/list.py
--- lollypop-0.9.42.orig/src/list.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/list.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/mpris.py lollypop-0.9.42/src/mpris.py
--- lollypop-0.9.42.orig/src/mpris.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/mpris.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # Copyright (c) 2013 Arnel A. Borja <kyoushuu@yahoo.com>
 # Copyright (c) 2013 Vadim Rutkovsky <vrutkovs@redhat.com>

diff -Naur lollypop-0.9.42.orig/src/notification.py lollypop-0.9.42/src/notification.py
--- lollypop-0.9.42.orig/src/notification.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/notification.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player.py lollypop-0.9.42/src/player.py
--- lollypop-0.9.42.orig/src/player.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player_base.py lollypop-0.9.42/src/player_base.py
--- lollypop-0.9.42.orig/src/player_base.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_base.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player_bin.py lollypop-0.9.42/src/player_bin.py
--- lollypop-0.9.42.orig/src/player_bin.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_bin.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player_externals.py lollypop-0.9.42/src/player_externals.py
--- lollypop-0.9.42.orig/src/player_externals.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_externals.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player_linear.py lollypop-0.9.42/src/player_linear.py
--- lollypop-0.9.42.orig/src/player_linear.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_linear.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player_queue.py lollypop-0.9.42/src/player_queue.py
--- lollypop-0.9.42.orig/src/player_queue.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_queue.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player_radio.py lollypop-0.9.42/src/player_radio.py
--- lollypop-0.9.42.orig/src/player_radio.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_radio.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player_rg.py lollypop-0.9.42/src/player_rg.py
--- lollypop-0.9.42.orig/src/player_rg.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_rg.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # Copyright (C) 2010 Jonathan Matthew (replay gain code)
 # This program is free software: you can redistribute it and/or modify

diff -Naur lollypop-0.9.42.orig/src/player_shuffle.py lollypop-0.9.42/src/player_shuffle.py
--- lollypop-0.9.42.orig/src/player_shuffle.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_shuffle.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/player_userplaylist.py lollypop-0.9.42/src/player_userplaylist.py
--- lollypop-0.9.42.orig/src/player_userplaylist.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/player_userplaylist.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/playlists.py lollypop-0.9.42/src/playlists.py
--- lollypop-0.9.42.orig/src/playlists.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/playlists.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_albums.py lollypop-0.9.42/src/pop_albums.py
--- lollypop-0.9.42.orig/src/pop_albums.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_albums.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_covers.py lollypop-0.9.42/src/pop_covers.py
--- lollypop-0.9.42.orig/src/pop_covers.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_covers.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_externals.py lollypop-0.9.42/src/pop_externals.py
--- lollypop-0.9.42.orig/src/pop_externals.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_externals.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_infos.py lollypop-0.9.42/src/pop_infos.py
--- lollypop-0.9.42.orig/src/pop_infos.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_infos.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_menu.py lollypop-0.9.42/src/pop_menu.py
--- lollypop-0.9.42.orig/src/pop_menu.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_menu.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_next.py lollypop-0.9.42/src/pop_next.py
--- lollypop-0.9.42.orig/src/pop_next.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_next.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_queue.py lollypop-0.9.42/src/pop_queue.py
--- lollypop-0.9.42.orig/src/pop_queue.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_queue.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_radio.py lollypop-0.9.42/src/pop_radio.py
--- lollypop-0.9.42.orig/src/pop_radio.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_radio.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_search.py lollypop-0.9.42/src/pop_search.py
--- lollypop-0.9.42.orig/src/pop_search.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_search.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/pop_tunein.py lollypop-0.9.42/src/pop_tunein.py
--- lollypop-0.9.42.orig/src/pop_tunein.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/pop_tunein.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/selectionlist.py lollypop-0.9.42/src/selectionlist.py
--- lollypop-0.9.42.orig/src/selectionlist.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/selectionlist.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/settings.py lollypop-0.9.42/src/settings.py
--- lollypop-0.9.42.orig/src/settings.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/settings.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/sync_mtp.py lollypop-0.9.42/src/sync_mtp.py
--- lollypop-0.9.42.orig/src/sync_mtp.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/sync_mtp.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/tagreader.py lollypop-0.9.42/src/tagreader.py
--- lollypop-0.9.42.orig/src/tagreader.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/tagreader.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/toolbar.py lollypop-0.9.42/src/toolbar.py
--- lollypop-0.9.42.orig/src/toolbar.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/toolbar.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/toolbar_end.py lollypop-0.9.42/src/toolbar_end.py
--- lollypop-0.9.42.orig/src/toolbar_end.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/toolbar_end.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/toolbar_infos.py lollypop-0.9.42/src/toolbar_infos.py
--- lollypop-0.9.42.orig/src/toolbar_infos.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/toolbar_infos.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/toolbar_playback.py lollypop-0.9.42/src/toolbar_playback.py
--- lollypop-0.9.42.orig/src/toolbar_playback.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/toolbar_playback.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/toolbar_title.py lollypop-0.9.42/src/toolbar_title.py
--- lollypop-0.9.42.orig/src/toolbar_title.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/toolbar_title.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/track.py lollypop-0.9.42/src/track.py
--- lollypop-0.9.42.orig/src/track.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/track.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/tunein.py lollypop-0.9.42/src/tunein.py
--- lollypop-0.9.42.orig/src/tunein.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/tunein.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/utils.py lollypop-0.9.42/src/utils.py
--- lollypop-0.9.42.orig/src/utils.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/utils.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/view.py lollypop-0.9.42/src/view.py
--- lollypop-0.9.42.orig/src/view.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/view.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/view_albums.py lollypop-0.9.42/src/view_albums.py
--- lollypop-0.9.42.orig/src/view_albums.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/view_albums.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/view_container.py lollypop-0.9.42/src/view_container.py
--- lollypop-0.9.42.orig/src/view_container.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/view_container.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/view_device.py lollypop-0.9.42/src/view_device.py
--- lollypop-0.9.42.orig/src/view_device.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/view_device.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/view_playlists.py lollypop-0.9.42/src/view_playlists.py
--- lollypop-0.9.42.orig/src/view_playlists.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/view_playlists.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/view_radios.py lollypop-0.9.42/src/view_radios.py
--- lollypop-0.9.42.orig/src/view_radios.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/view_radios.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/widgets_album.py lollypop-0.9.42/src/widgets_album.py
--- lollypop-0.9.42.orig/src/widgets_album.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/widgets_album.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/widgets_device.py lollypop-0.9.42/src/widgets_device.py
--- lollypop-0.9.42.orig/src/widgets_device.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/widgets_device.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/widgets_playlist.py lollypop-0.9.42/src/widgets_playlist.py
--- lollypop-0.9.42.orig/src/widgets_playlist.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/widgets_playlist.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/widgets_radio.py lollypop-0.9.42/src/widgets_radio.py
--- lollypop-0.9.42.orig/src/widgets_radio.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/widgets_radio.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/widgets_rating.py lollypop-0.9.42/src/widgets_rating.py
--- lollypop-0.9.42.orig/src/widgets_rating.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/widgets_rating.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/widgets_track.py lollypop-0.9.42/src/widgets_track.py
--- lollypop-0.9.42.orig/src/widgets_track.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/widgets_track.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/wikipedia.py lollypop-0.9.42/src/wikipedia.py
--- lollypop-0.9.42.orig/src/wikipedia.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/wikipedia.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

diff -Naur lollypop-0.9.42.orig/src/window.py lollypop-0.9.42/src/window.py
--- lollypop-0.9.42.orig/src/window.py 2015-07-24 15:59:25.000000000 +0300
+++ lollypop-0.9.42/src/window.py      2015-07-29 10:55:10.357181067 +0300
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # Copyright (c) 2014-2015 Cedric Bellegarde <cedric.bellegarde@adishatz.org>
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by

