diff -Naur lollypop-0.9.42.orig/data/lollypop.appdata.xml.in lollypop-0.9.42/data/lollypop.appdata.xml.in
--- lollypop-0.9.42.orig/data/lollypop.appdata.xml.in	2015-07-03 17:07:26.000000000 +0300
+++ lollypop-0.9.42/data/lollypop.appdata.xml.in	2015-07-29 11:49:40.618667871 +0300
@@ -6,11 +6,30 @@
  <_summary>Play and organize your music collection</_summary>
  <description>
   <_p>
-   Music player for the GNOME desktop
+	Lollypop is a new GNOME music playing application.
+  </_p>
+  <_p>
+	It provides:
+		- mp3/4, ogg and flac.
+		- Genre/Cover browsing.
+		- Genre/Artist/Cover browsing.
+		- Search.
+		- Main playlist (called queue in other apps).
+		- Party mode.
+		- Replay gain.
+		- Cover art downloader.
+		- Context artist view.
+		- MTP sync.
+		- Fullscreen view.
+		- Radios support.
+		- Last.fm support.
+		- Auto install codecs.
+		- HiDPI support.
+		- Tunein support.
   </_p>
  </description>
  <screenshots>
-  <screenshot type="default" width="750" height="469">https://raw.githubusercontent.com/gnumdk/lollypop/gh-pages/images/lollypop1.png</screenshot>
+  <screenshot type="default" width="800" height="439">https://raw.githubusercontent.com/gnumdk/lollypop-data/master/main_small.jpg</screenshot>
  </screenshots>
  <url type="homepage">https://github.com/gnumdk/lollypop</url>
  <updatecontact>cedric.bellegarde@adishatz.org</updatecontact>

